website/src/components/misc/winking-logo.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from "react";
+import { motion, SVGMotionProps } from "framer-motion";
+
+const transition = {
+  duration: 1.1,
+  times: [0, 0.35, 0.6, 0.8, 0.9, 1],
+};
+
+const glassesStages = [
+  "m 612.43331,842.49922 c 0,0 61.66627,-7.5 72.36627,-7.5 0.052,0.47569 0.1524,0.94481 0.3,1.4 10.6,33.6 10.4,71.1 26.8,102.8 6.5,12.5 15.3,26.9 28.3,33.8 30,16 163.2,19.8 192.7,4.7 43.3,-22.2 75.20002,-138.2 75.40002,-141.7 8.3,0 77.3,0.1 85.6,0 0.2,3.5 32.1,119.5 75.4,141.7 29.5,15.2 162.7,11.3 192.7,-4.7 13,-6.9 21.8,-21.3 28.3,-33.8 16.4,-31.7 16.2,-69.2 26.8,-102.8 0.1476,-0.45519 0.2481,-0.92431 0.3,-1.4 12.4,0.1 61.1363,7.08726 75.8363,9.18726 0,0 -1.485,-67.6816 -3.8605,-69.74787 -2.6797,-2.33083 -190.7667,-4.6886 -200.4317,-5.44994 -6.7531,-0.53196 -173.1086,1.83851 -204.4058,2.88867 -23.1742,0.7776 -56.8563,0.93105 -64.7366,0.78641 -8.3223,-0.15275 -138.07252,-4.98422 -207.58982,-4.33973 -48.731,0.45178 -185.3393,6.29683 -185.3393,6.29683 -1.2285,2.80403 -12.24427,17.43256 -14.43887,67.87837 z",
```

Comment:
Is not it possible to do the animation in within the SVG file?

Reply:
you mean with svg animations? 
I just used framer for convenience. really not fit on svg animations.